backend/server/server.js

```diff
@@ -43,7 +43,7 @@ const getMainComponent = (url, payload) => {
     return (
         <Loadable.Capture report={(moduleName) => modules.push(moduleName)}>
             <StaticRouter location={url} context={context}>
-                <App {...payload.layout} {...payload.viewAs} {...payload.page} />
+                <App layout={payload.layout} {...payload.viewAs} {...payload.page} />
```

Comment:
are you sure this is needed?

Reply:
Yes... because inside we're doing `props.layout` and because `page.layout` have inside a `layout` property (normally in null) it wasn't breaking anything but it was rendering the pages first with the alert and then rerender with the content, this way, we make sure that when we do in `<App />` `this.props.layout` we're getting the layout object and not the `layout.layout` property.